india_compliance/gst_india/client_scripts/sales_invoice.js

```diff
@@ -26,5 +26,10 @@ frappe.ui.form.on(DOCTYPE, {
 
     before_submit(frm) {
         frm.doc._submitted_from_ui = 1;
-    }
+    },
+
+    refresh(frm) {
+        frm.set_df_property("port_code", "ignore_validation", 1);
```

Comment:
Did you try onload?
Also, `set_df_property` can be part of set_port_code_options